### ISSUE TITLE
Misspelt Liberia

### ISSUE THREAD

**Issue by Thomas-Upfield** [ClearContracts/cardano-constitution-voting-app#206]
After a delegate pointed this out to me I realised in the spreadsheet of delegate information that I sent to you I misspelt Liberia as Libeira. 

Please could you amend this in the records as "Liberia"? 

Thanks and sorry for the cock-up.

**Comment by mattlaux** [comment]
Yep we'll make that change. I believe there is also a typo with "USA, Fort Mayers". It should be "USA, Fort Myers". I'll make that change as well.
